x-pack/plugins/upgrade_assistant/server/saved_object_types/reindex_operation_saved_object_type.ts

```diff
@@ -30,10 +42,16 @@ export const reindexOperationSavedObjectType: SavedObjectsType = {
         type: 'date',
       },
       lastCompletedStep: {
-        type: 'integer',
+        type: 'long',
       },
       errorMessage: {
-        type: 'keyword',
+        type: 'text',
+        fields: {
+          keyword: {
+            type: 'keyword',
+            ignore_above: 256,
+          },
+        },
       },
       reindexTaskPercComplete: {
         type: 'float',
```

Comment:
This was originally being dynamically mapped as `long` but I believe that was an error, since we expect floating point values where this field is consumed.